Dockerfile.dev

```diff
@@ -40,6 +40,7 @@ RUN SETUPTOOLS_SCM_PRETEND_VERSION_FOR_FLYTEKIT=$PSEUDO_VERSION \
             -e /flytekit \
             -e /flytekit/plugins/flytekit-deck-standard \
             -e /flytekit/plugins/flytekit-flyteinteractive \
+            obstore==0.3.0b9 \
```

Comment:
is this needed or can we put it in a dev requirements file?  if we need it, should we bump to 0.6.0?